FreeRTOS/Test/CMock/smp/Makefile

```diff
@@ -5,12 +5,15 @@ MAKEFILE_ABSPATH     := $(abspath $(lastword $(MAKEFILE_LIST)))
 include ../makefile.in
 
 # SUITES lists the suites contained in subdirectories of this directory
-SUITES	+=	single_priority_no_timeslice
+
 SUITES	+=	single_priority_timeslice
 SUITES	+=	multiple_priorities_no_timeslice
 SUITES	+=	single_priority_no_timeslice_covg
 SUITES	+=	single_priority_timeslice_thread
 SUITES	+=	multiple_priorities_no_timeslice_covg
+SUITES	+=	single_priority_no_timeslice
+SUITES	+=	multiple_priorities_no_timeslice_divya
```

Comment:
Create the test group ( folder ) only when FreeRTOSConfig.h is different. The test group naming should reflect the difference in FreeRTOSConfig.h.
Can we add the test cases in multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice.c?